react-debugging.html

```diff
@@ -63,22 +63,24 @@
           // No need to change anything ABOVE this line ^
 
```

Comment:
The object of this section was to debug the react app by inspecting the page and following the error messages on the console. By the 3rd error message correction you should see the cards. I think you should be able to see cards by the method you used here. However, they will appear differently.